src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -228,6 +229,7 @@ trait GraphDAO {
           val workspaceVertex = getSinglePipelineResult(workspacePipeline(graph, workspaceName.namespace, workspaceName.name)).get
           new GremlinPipeline(vertex).out(workflowEdge).linkIn(workflowEdge, workspaceVertex).iterate()
         case Some(x: WorkflowFailure) => serializeDomainObjects(vertex, seq.asInstanceOf[Seq[WorkflowFailure]], workflowFailureEdge, (wf: WorkflowFailure) => (wf.entityType, wf.entityName), graph, workspaceName)
+        case Some(a: Attribute) => serializeSeq(graph, workspaceName, vertex, key, seq.asInstanceOf[Seq[Attribute]])
```

Comment:
I know it's not really part of this PR but I'd like to register my displeasure at the serialization of `Seq[]` here. This is not very polymorphic-collectiony. Whatever happened to `seq.map( _.serialize )` ?